DOCS-1696: Getting Started MVD (#1017)
MultiSafepay/docs: content/guides/getting-started/_index.md

---
title: 'Getting Started'
breadcrumb_title: 'Getting Started'
weight: 10
meta_title: "Getting Started - MultiSafepay Docs"
meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
read_more: "."
logo: '/svgs/Getting started.svg'
layout: 'single'
weight: 10
short_description: "Follow our step-by-step instructions from creating a 
test account to processing your first payment."

---

Welcome to MultiSafepay. We break down barriers and remove obstacles for commerce by providing smart payment solutions. Our core services allow you to accept payments, manage transactions and stimulate online sales. On top of that, we provide a suite of tools to boost conversion rates, create the perfect checkout experience and gain financial insights.

> With this guide, you will process your first payment in five easy steps.

Reviewer: By following this guide, you will be able to process your first payment in five easy steps.

> With this guide, you will process your first payment in five easy steps.

Author: You can start processing payments in five easy steps:

> With this guide, you will process your first payment in five easy steps.

Author: ```suggestion
You can start processing payments in five easy steps:
```

{{< br >}}

## Step 1: Create a test account
To create a test account, follow these steps:
1. Fill out the signup form to [create a free test account](https://testmerchant.multisafepay.com/signup)

> 2. Open the e-mail containing your **security code**

Reviewer: e-mail = email
As mentioned in the EN Woordenlijst on Confluence

> 2. Open the e-mail containing your **security code**

Author: ```suggestion
2. Open the email containing your **security code**
```

> 3. [Log in](https://testmerchant.multisafepay.com/) using the **security code** provided in the e-mail

Reviewer: e-mail = email

> 3. [Log in](https://testmerchant.multisafepay.com/) using the **security code** provided in the e-mail

Author: ```suggestion
3. [Log in](https://testmerchant.multisafepay.com/) using the **security code** provided in the email
```

    {{< button href="https://docs.multisafepay.com/integrations/wrappers/" header="Wrappers & SDK's" text="Wrappers & SDK's facilitate development and testing for web and mobile applications" img="/svgs/Hosted_plugins.svg" alt="Hosted Plugins icon" >}}
{{< /buttons >}}

{{< br >}}

### Other integration methods
* [**Cash register software**](/integrations/cash-register-software/): simplify the order process for restaurants, cafes, hotels, and establishments alike
* [**Reservation Management**](/integrations/reservation-management/): allow your customers to make reservations and pay online
* [**Credit & Subscription Management**](/integrations/credit-and-subscription-management/): manage your subscriptions and collect your subscription fees

{{< br >}}

*Are you not sure which integration method best suits your business?  
Our [integration specialists](mailto:integration@multisafepay.com) are happy to help you.*

{{< br >}}

## Step 4: Go live!

> When you are ready, you [sign up for a live account](https://merchant.multisafepay.com/signup) to start processing real transactions.

Reviewer: When you are ready, you can [sign up for a live account](https://merchant.multisafepay.com/signup) to begin processing real transactions.

> When you are ready, you [sign up for a live account](https://merchant.multisafepay.com/signup) to start processing real transactions.

Author: ```suggestion
When you are ready, you [sign up for a live account](https://merchant.multisafepay.com/signup) to begin processing real transactions.
```

> Because we are a [payment service provider](/faq/getting-started/glossary/#payment-service-provider-psp) and [acquirer](/faq/getting-started/glossary/#acquirer), we are legally required to perform extra checks, which require you to submit multiple forms. You can prepare [these documents](/faq/getting-started/how-do-i-know-which-documents-i-need-to-submit-and-which-data-to-post-on-my-website) in advance if you wish to speed up the onboarding process.

Reviewer: MultiSafepay is a [payment service provider](/faq/getting-started/glossary/#payment-service-provider-psp) and [acquirer](/faq/getting-started/glossary/#acquirer), therefore, we are legally required to perform extra checks, which require you to submit multiple forms. You can prepare [these documents](/faq/getting-started/how-do-i-know-which-documents-i-need-to-submit-and-which-data-to-post-on-my-website) in advance if you wish to speed up the onboarding process.

> Because we are a [payment service provider](/faq/getting-started/glossary/#payment-service-provider-psp) and [acquirer](/faq/getting-started/glossary/#acquirer), we are legally required to perform extra checks, which require you to submit multiple forms. You can prepare [these documents](/faq/getting-started/how-do-i-know-which-documents-i-need-to-submit-and-which-data-to-post-on-my-website) in advance if you wish to speed up the onboarding process.

Author: ```suggestion
MultiSafepay is a [payment service provider](/faq/getting-started/glossary/#payment-service-provider-psp) and [acquirer](/faq/getting-started/glossary/#acquirer). Therefore, we are legally required to perform extra checks, which require you to submit multiple forms. You can [prepare these documents](/faq/getting-started/how-do-i-know-which-documents-i-need-to-submit-and-which-data-to-post-on-my-website) in advance if you wish to speed up the onboarding process.
```

> *If you need help creating a live account, please reach out to our [sales team](mailto:sales@multisafepay.com)*

Reviewer: *If you require assistance creating a live account, please reach out to our [sales team](mailto:sales@multisafepay.com)*

> *If you need help creating a live account, please reach out to our [sales team](mailto:sales@multisafepay.com)*

Author: ```suggestion
*If you require assistance creating a live account, please reach out to our [sales team](mailto:sales@multisafepay.com)*
```
